wemake_python_styleguide/visitors/cst/attributes.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from libcst import (
```

Comment:
Let's stick to the same convention as we use in `ast`.
We do `import ast`.

So, let's use `import libcst` instead

Reply:
done